docs/interfaces/postgres.rst

```diff
@@ -190,9 +190,11 @@ table available in CrateDB::
     | 1186 | interval                     |     1187 |       0 |     16 | T           |
     | 1187 | _interval                    |        0 |    1186 |     -1 | A           |
     | 2249 | record                       |     2287 |       0 |     -1 | P           |
+    | 2276 | any                          |        0 |       0 |      4 | P           |
+    | 2277 | _any                         |        0 |    2276 |     -1 | A           |
```

Comment:
in PostgreSQL this is `anyarray`. why is it `_any`? this is not very expressive ;)

Reply:
right, will adjust it. I am wondering why other array types represented as `_type` and the array of any type as `anyarray`. maybe it is a special case ..